README.md

```diff
@@ -244,6 +243,29 @@ The remote dataset url is taken from the Data URL section of an OPeNDAP endpoint
 $ compliance-checker --test=acdd:1.3 "http://sos.maracoos.org/stable/dodsC/hrecos/stationHRMARPH-agg.ncml"
 ```
 
+### Checking against remote ERDDAP Datasets
+
+ERDDAP datasets are becoming a popular way to access data. Supply an ERDDAP `TableDAP` or `GridDAP` URL to the checker:
+
+```shell
+$ compliance-checker --test ioos:1.2 "https://pae-paha.pacioos.hawaii.edu/erddap/griddap/pibhmc_bathy_60m_guam"
+```
+
+Ensure to supply the URL *without* the format extension at the end (no `.nc`, `.ncCF`, etc.).
+
+Some examples of ERDDAP datasets:
+
+  - https://pae-paha.pacioos.hawaii.edu/erddap/info/AWS-HIMB/index.html
+  - http://erddap.secoora.org/erddap/tabledap/edu_usf_marine_comps_1407d550.html
+  - http://erddap.cencoos.org/erddap/tabledap/bodega-bay-bml_wts.html
+  - http://erddap.cencoos.org/erddap/tabledap/fort-point.html
+  - http://erddap.cencoos.org/erddap/tabledap/edu_humboldt_humboldt.html
+  - http://erddap.cencoos.org/erddap/tabledap/edu_calpoly_marine_morro.html
+  - http://erddap.cencoos.org/erddap/tabledap/mlml_mlml_sea.html
+  - http://erddap.cencoos.org/erddap/tabledap/mlml_mlml_met.html
+  - http://erddap.cencoos.org/erddap/tabledap/mlml_monterey.html
+  - http://erddap.cencoos.org/erddap/tabledap/edu_humboldt_tdp.html
+
```

Comment:
I would leave out the .html on all of these.  ERDDAP is smart enough that if you use the extension-less URL, it will display the HTML view.  As it is, it's kind of sending mixed signals for how to use.

Reply:
Should be all set now 👍

Reply:
Oops, didn't see the PacIOOS one actually needs to be:  `https://pae-paha.pacioos.hawaii.edu/erddap/tabledap/AWS-HIMB`

Reply:
Ah yes... it would be good to point to a real dataset!

Reply:
#fakenews